test/turnt.toml

```diff
@@ -6,3 +6,10 @@ command = "odgi view -g -i {filename} | python3 ../slow_odgi/mygfa.py"
 binary = true
 command = "python3 ../slow_odgi/mygfa.py < {filename}"
 
+[envs.crush_oracle]
+binary = true
+command = "odgi crush -i {filename} --out=crushed.og; odgi view  -g -i crushed.og | python3 ../slow_odgi/mygfa.py"
```

Comment:
I don't love that I have to create a temporary file. Among other issues, turnt actually _runs_ on `crushed.og`, which is silly. Piping would be nicer, but `{filename} | odgi view -g -i` does not work.